src/khoj/utils/helpers.py

```diff
@@ -287,6 +290,7 @@ class ConversationCommand(str, Enum):
     ConversationCommand.General: "Use this when you can answer the question without any outside information or personal knowledge",
     ConversationCommand.Notes: "To search the user's personal knowledge base. Especially helpful if the question expects context from the user's notes or documents.",
     ConversationCommand.Online: "To search for the latest, up-to-date information from the internet. Note: **Questions about Khoj should always use this data source**",
+    ConversationCommand.Webpage: "To use if the user has directly provided the webpage urls or you are certain of the webpage urls to read.",
```

Comment:
Good to unify the language here. I think `ConversationCommand.General` should be updated to match the `To <verb> if <blah>` pattern.